src/router/db.ts

```diff
@@ -76,17 +78,29 @@ db.get('/remove/:dbAlias', async (req, res) => {
 
 db.post('/apply', async (req, res) => {
   const { dbAlias, dryRun } = req.body;
+  if (!dbAlias) return res.status(400).json(
+    `Required key(s) not provided: ${[
+      'dbAlias',
+    ].filter(k => !req.body.hasOwnProperty(k)).join(', ')}`
+  );
```

Comment:
If there's only one key, do we need the array and filter stuff?

Reply:
that is true